charts/cluster-api-runtime-extensions-nutanix/templates/kube-vip/manifests/kube-vip-configmap.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.hooks.kubeVip.defaultTemplateConfigMap.create }}
+apiVersion: v1
+data:
+  kube-vip.yaml: |
```

Comment:
Can we generate the content of this using https://kube-vip.io/docs/installation/daemonset/#creating-the-manifest instead of just hard-coding them in here? Might make upgrades easier and hopefully not miss anything.

Reply:
Pushed https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/562 to do that, let me know what you think.